test/functional/rpc_packages.py

```diff
@@ -158,156 +148,116 @@ def test_chain(self):
         node = self.nodes[0]
-        coin = self.coins.pop()
-        inputs = [{"txid": coin["txid"], "vout": 0, "sequence": MAX_BIP125_RBF_SEQUENCE}]
+
+        coin = self.wallet.get_utxo()
         fee = Decimal('0.00125000')
-        output = {node.get_deterministic_priv_key().address: 50 - fee}
-        raw_replaceable_tx = node.createrawtransaction(inputs, output)
-        signed_replaceable_tx = node.signrawtransactionwithkey(hexstring=raw_replaceable_tx, privkeys=self.privkeys)
-        testres_replaceable = node.testmempoolaccept([signed_replaceable_tx["hex"]])
-        replaceable_tx = tx_from_hex(signed_replaceable_tx["hex"])
+        replaceable_tx = self.wallet.create_self_transfer(utxo_to_spend=coin, sequence=MAX_BIP125_RBF_SEQUENCE, fee = fee)
+        testres_replaceable = node.testmempoolaccept([replaceable_tx["hex"]])
         assert_equal(testres_replaceable, [
-            {"txid": replaceable_tx.rehash(), "wtxid": replaceable_tx.getwtxid(),
-            "allowed": True, "vsize": replaceable_tx.get_vsize(), "fees": { "base": fee }}
+            {"txid": replaceable_tx["txid"], "wtxid": replaceable_tx["wtxid"],
+            "allowed": True, "vsize": replaceable_tx['tx'].get_vsize(), "fees": { "base": fee }}
         ])
 
-        # Replacement transaction is identical except has double the fee
-        replacement_tx = tx_from_hex(signed_replaceable_tx["hex"])
-        replacement_tx.vout[0].nValue -= int(fee * COIN)  # Doubled fee
-        signed_replacement_tx = node.signrawtransactionwithkey(replacement_tx.serialize().hex(), self.privkeys)
-        replacement_tx = tx_from_hex(signed_replacement_tx["hex"])
-
-        self.log.info("Test that transactions within a package cannot replace each other")
-        testres_rbf_conflicting = node.testmempoolaccept([signed_replaceable_tx["hex"], signed_replacement_tx["hex"]])
+        replacement_tx = self.wallet.create_self_transfer(utxo_to_spend=coin, sequence=MAX_BIP125_RBF_SEQUENCE, fee = 2 * fee)
+        testres_rbf_conflicting = node.testmempoolaccept([replaceable_tx["hex"], replacement_tx["hex"]])
         assert_equal(testres_rbf_conflicting, [
-            {"txid": replaceable_tx.rehash(), "wtxid": replaceable_tx.getwtxid(), "package-error": "conflict-in-package"},
-            {"txid": replacement_tx.rehash(), "wtxid": replacement_tx.getwtxid(), "package-error": "conflict-in-package"}
+            {"txid": replaceable_tx["txid"], "wtxid": replaceable_tx["wtxid"], "package-error": "conflict-in-package"},
+            {"txid": replacement_tx["txid"], "wtxid": replacement_tx["wtxid"], "package-error": "conflict-in-package"}
         ])
 
         self.log.info("Test that packages cannot conflict with mempool transactions, even if a valid BIP125 RBF")
-        node.sendrawtransaction(signed_replaceable_tx["hex"])
-        testres_rbf_single = node.testmempoolaccept([signed_replacement_tx["hex"]])
-        # This transaction is a valid BIP125 replace-by-fee
```

Comment:
Why delete the comment?

Reply:
Was inadvertently. Done in 17cad448516a6906ff637593ab57df332fade5d2.